index.html

```diff
@@ -11,21 +11,28 @@
 </head>
 
 <body>
+   
+
     <div class="wrapper">
-        <nav class="nav-bar-desktop">
-            <a class="logo" href="#">Hamma Azarok</a>
-            <ul class="nav-list">
-                <li class="nav-item"><a href="#portfolio">Portofolio</a></li>
-                <li class="nav-item"><a href="#about">About</a></li>
-                <li class="nav-item"><a href="#contact">Contact</a></li>
+    <!-- Create the Navigation menu this way -->
+    <header class="nav-header">
+        <a href="#" class="nav-log">
+            <img src="img/logo.png" />
+        </a>
+        <nav id="navbar">
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="#" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="#portfolio" class="nav-link">Portofolio</a></li>
+                <li class="nav-item"><a href="#about" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
             </ul>
-        </nav>
-        <header class="nav-bar">
-            <a href="#">
-                <p>Hamma Azarok</p>
+            <a href="#" class="hamburger">
+                <span class="src-only" hidden="true">Open  main menu</span>
+                <i class="fas fa-bars"></i>
             </a>
-            <nav> <img src="img/Enabled.png" alt="Logo"></nav>
-        </header>
+        </nav>
+    </header>
+    
         <section class="header">
             <div class="header-content">
                 <h1 class="header-text">I’m Hamma
```

Comment:
Please stick to the design and make your background blur as of the design when the mobile menu appears.

| Your Design | Figma Design |
|-------------|---------------|
|<img width="311" alt="image" src="https://user-images.githubusercontent.com/34578631/173549831-715ad029-0d1e-4378-b0bb-ab2d3f1215a2.png"> | <img width="374" alt="image" src="https://user-images.githubusercontent.com/34578631/173549899-779fe3bc-e87f-4b82-b970-7b177042ee8e.png"> |